auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/service/TokenService.java

```diff
@@ -86,21 +87,21 @@ public Cookie generateJwtCookieForOAuth2User(OAuth2UserInfo oAuth2UserInfo) {
         String lastName = oAuth2UserInfo.getLastName();
         CustomUserDetails customUserDetails = new CustomUserDetails(email, "", firstName, lastName, Collections.emptyList());
 
-        String jwt = generateToken(new UsernamePasswordAuthenticationToken(customUserDetails, ""));
+        String jwt = generateToken(new UsernamePasswordAuthenticationToken(customUserDetails, ""), false);
         Cookie cookie = new Cookie(SOCIAL_MEDIA_COOKIE_NAME, jwt);
         cookie.setPath("/");
         cookie.setMaxAge(SOCIAL_MEDIA_COOKIE_MAX_AGE);
         return cookie;
     }
 
-    public String generateToken(Authentication authentication) {
+    public String generateToken(Authentication authentication, boolean rememberMe) {
```

Comment:
We could also send token expiration time as parameter. We could use some values from enum where all expiration times are stored based on token type.